playbook/app/pb_kits/playbook/pb_table/docs/_table_sm.jsx

```diff
@@ -1,12 +1,22 @@
-import React from 'react'
+/*
+I tried 4 options to pass this file as a string.
+- We could use "source" from Rails as Jasper did here: https://github.com/powerhome/playbook/commit/5f01b2fcaff67c5b2944a799908681a1e1db6bc5#diff-9933dfd1f79c7c87374bd19e7575b231da08f834b3360d7c02e6dc1d0fd5d100
+- We could use MDX lib to serialize the code: https://github.com/hashicorp/next-mdx-remote
+- We could use raw-loader from Asset Modules: https://webpack.js.org/guides/asset-modules/
+- Or we could just export the string in the file as I did below.
 
-import Table from '../_table'
+But even so, in the end, we would have to change the document a little bit to turn it into "real code" without errors.
 
-const TableSm = (props) => {
+For example: for this one, I had to remove the parameter "props" (because we were not using it) and change the way that we import the component Table from Playbook.
+*/
+const TableSmCode = `import React from 'react'
```

Comment:
We can't require our examples to be strings to display. Developers wouldn't be able to reuse the code in their environments